app/controllers/helper/ShapefilesCreatorHelper.java

```diff
@@ -0,0 +1,443 @@
+                }
+            }));
+            featureBuilder.add(l.labelTemporary());
+            featureBuilder.add(l.neighborhoodName());
+            featureBuilder.add(l.gsvPanoramaId());
+            featureBuilder.add(l.heading());
+            featureBuilder.add(l.pitch());
+            featureBuilder.add(l.zoom());
+            featureBuilder.add(l.canvasX());
+            featureBuilder.add(l.canvasY());
+            featureBuilder.add(l.canvasWidth());
+            featureBuilder.add(l.canvasHeight());
+            SimpleFeature feature = featureBuilder.buildFeature(null);
+            features.add(feature);
+        }
+
+
+        createGeneralShapeFile(outputFile, TYPE, features);
+    }
+
+    public static void createStreetShapefile(String outputFile, List<StreetAttributeSignificance> streets) throws Exception{
+        /*
+         * We use the DataUtilities class to create a FeatureType that will describe the data in our
+         * shapefile.
+         *
+         * See also the createFeatureType method below for another, more flexible approach.
+         */
+        final SimpleFeatureType TYPE =
+                DataUtilities.createType(
+                        "Location",
+                        "the_geom:LineString:srid=4326,"
+                                + // <- the geometry attribute: Line type
+                                "streetId:Integer,"
+                                + // <- StreetId
+                                "score:Double," 
+                                + // street score
+                                "sigRamp:Double,"
+                                + // curb ramp significance score
+                                "sigNoRamp:Double,"
+                                + // no Curb ramp significance score
+                                "sigObs:Double,"
+                                + // obstacle significance score
+                                "sigSurfce:Double," 
+                                + // Surface problem significance score
+                                "nRamp:Double,"
+                                + // curb ramp feature score
+                                "nNoRamp:Double,"
+                                + // no Curb ramp feature score
+                                "nObs:Double,"
+                                + // obstacle feature score
+                                "nSurfce:Double" // Surface problem feature score
+                );
+
+
+
+        /*
+         * A list to collect features as we create them.
+         */
+        List<SimpleFeature> features = new ArrayList<>();
+
+        /*
+         * GeometryFactory will be used to create the geometry attribute of each feature,
+         * using a Point object for the location.
+         */
+        GeometryFactory geometryFactory = JTSFactoryFinder.getGeometryFactory();
+
+        SimpleFeatureBuilder featureBuilder = new SimpleFeatureBuilder(TYPE);
+
+        // for(Street s : streets){
```

Comment:
is this no longer necessary? should it be deleted?

Reply:
yeah i forgot to remove it after overhauling the data structures